docs/cloud-native-security/cspm-get-started-aws.asciidoc

```diff
@@ -1,14 +1,12 @@
 [[cspm-get-started]]
-= Get started with CSPM
+= Get started with CSPM for AWS
```

Comment:
@benironside I see we haven't added the aws org onboarding steps yet, can we do that? Check the thread below for what this process looks like:

https://elastic.slack.com/archives/C05EPKJ5XSL/p1694156910752999?thread_ts=1694054229.006979&cid=C05EPKJ5XSL